include/re_http.h

```diff
@@ -169,6 +169,9 @@ int http_client_set_cert(struct http_cli *cli, const char *path);
 int http_client_set_certpem(struct http_cli *cli, const char *pem);
 int http_client_set_key(struct http_cli *cli, const char *path);
 int http_client_set_keypem(struct http_cli *cli, const char *pem);
+int http_client_use_chain(struct http_cli *cli, const char *path);
+int http_client_use_chainpem(struct http_cli *cli, const char *chain,
+			     int len_chain);
```

Comment:
Please use `size_t len_chain` which is consistant with the API ...

Reply:
I wanted to prevent the cast from `size_t` to `int` when calling `BIO_new_mem_buf`, but consistency makes sense. Done.